### PR TITLE
[SYCL][libdevice] Move imf abs to a separate file

### PR DESCRIPTION
**Pull request by jinge90**

### CODE REVIEW

**Comment by jinge90** [comment]
Hi, @akolesov-intel and @zettai-reido 
This pr moves "abs" to a separate file, all functions required by pytorch will be put there and we will use it to build a new spv file for pytorch.
Thanks very much.

File: libdevice/imf/imf_fp32_dl.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//==--------------------------- imf_fp32_dl.cpp -  -------------------------==//
```
Comment (dm-vodopyanov) [inline libdevice/imf/imf_fp32_dl.cpp:1]: Some description after `imf_fp32_dl.cpp -` looks missed

Comment (dm-vodopyanov) [reply libdevice/imf/imf_fp32_dl.cpp:1]: Or closer - see the 1st line in some random file in `libdevice/imf` folder: https://github.com/intel/llvm/blob/sycl/libdevice/imf/imf_inline_bf16.cpp#L1

**Review by xtian-github — APPROVED** [review]
LGTM

**Review by xtian-github — APPROVED** [review]
LGTM

**Review by akolesov-intel — APPROVED** [review]

File: libdevice/imf/imf_fp32_dl.cpp (hunk 1 above)
Comment (jinge90) [reply libdevice/imf/imf_fp32_dl.cpp]: Hi, @dm-vodopyanov 
Added the small description, could you help review again?
Thanks very much.

Comment (dm-vodopyanov) [reply libdevice/imf/imf_fp32_dl.cpp]: @jinge90 there should be a single line which starts with `//==` and ends with `==//`
https://llvm.org/docs/CodingStandards.html#file-headers

Comment (jinge90) [reply libdevice/imf/imf_fp32_dl.cpp]: Hi, @dm-vodopyanov 
Just adjusted the file header.
Thanks very much.

**Review by dm-vodopyanov — APPROVED** [review]

**Pull request merged**